automation/src/test/scala/org/broadinstitute/dsde/firecloud/BillingSpec.scala

```diff
@@ -19,18 +22,99 @@ class BillingSpec extends FreeSpec with WebBrowserSpec with CleanUp
         signIn(Config.Users.owner)
 
         val billingPage = new BillingManagementPage().open
-        val projectName = "billing-spec-create-" + makeRandomId()
-        logger.info(s"Creating billing project: $projectName")
+        val billingProjectName = "billing-spec-create-" + makeRandomId() // is this a unique ID?
```

Comment:
No, it's not guaranteed to be unique. Unfortunately, the max length of a Google project name is too short to allow us to use a UUID. `makeRandomId()` defaults to 7 characters/digits which will hopefully be unique enough for the foreseeable future. If it becomes a problem, I think we have room to increase the length to 8 or 9.